cmake/modules/BuildOpenTracing.cmake

```diff
@@ -19,7 +19,7 @@ function(build_opentracing)
   else()
     set(make_cmd ${CMAKE_COMMAND} --build <BINARY_DIR> --target opentracing)
   endif()
-  set(install_cmd ${CMAKE_MAKE_PROGRAM} install)
+  set(install_cmd DESTDIR= ${CMAKE_MAKE_PROGRAM} install)
```

Comment:
please note, the linked build log will be removed after a while, better off pasting the related log message in the commit message.